ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjcxNDExMzI2My4xNzQ2MzE0MjI2In0sImFub25JZCI6IjU1Yjg2NGI3OGYxMGQzY2MifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoubHUvZnIvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI1NCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjEzNDgzNDM0NzYyMzc3MzY2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55975e3bf75a <unknown>
#1 0x55975de724b0 <unknown>
#2 0x55975de58d86 <unknown>
#3 0x55975de5769a <unknown>
#4 0x55975de57dff <unknown>
#5 0x55975de57d54 <unknown>
#6 0x55975de7669f <unknown>
#7 0x55975df1b92a <unknown>
#8 0x55975dee9592 <unknown>
#9 0x55975df0f5e0 <unknown>
#10 0x55975dee9363 <unknown>
#11 0x55975deb5d63 <unknown>
#12 0x55975deb69c1 <unknown>
#13 0x55975e384a6b <unknown>
#14 0x55975e388951 <unknown>
#15 0x55975e36cb62 <unknown>
#16 0x55975e3894c4 <unknown>
#17 0x55975e35113f <unknown>
#18 0x55975e3ad6f8 <unknown>
#19 0x55975e3ad8d6 <unknown>
#20 0x55975e3be5a6 <unknown>
#21 0x7f200069caa4 <unknown>
#22 0x7f2000729c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNzE0MTEzMjYzLjE3NDYzMTQyMjYifSwiYW5vbklkIjoiNTViODY0Yjc4ZjEwZDNjYyJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5sdS9mci9wYXNzZW5nZXJjYXJzL21vZGVscy9zdXYvZ2xjL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6NH0sIl90b29sc0V2ZW50TGlua0lkIjoiNTk0NjY5MDQ2OTQwNzgxMyIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55975e3bf75a <unknown>
#1 0x55975de724b0 <unknown>
#2 0x55975de58d86 <unknown>
#3 0x55975de5769a <unknown>
#4 0x55975de57dff <unknown>
#5 0x55975de57d54 <unknown>
#6 0x55975de7669f <unknown>
#7 0x55975df1b92a <unknown>
#8 0x55975dee9592 <unknown>
#9 0x55975df0f5e0 <unknown>
#10 0x55975dee9363 <unknown>
#11 0x55975deb5d63 <unknown>
#12 0x55975deb69c1 <unknown>
#13 0x55975e384a6b <unknown>
#14 0x55975e388951 <unknown>
#15 0x55975e36cb62 <unknown>
#16 0x55975e3894c4 <unknown>
#17 0x55975e35113f <unknown>
#18 0x55975e3ad6f8 <unknown>
#19 0x55975e3ad8d6 <unknown>
#20 0x55975e3be5a6 <unknown>
#21 0x7f200069caa4 <unknown>
#22 0x7f2000729c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyNTQ2IiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJmcl9MVSIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2ZyL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNTQifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjcxNDExMzI2My4xNzQ2MzE0MjI2In0sImFub25JZCI6IjU1Yjg2NGI3OGYxMGQzY2MifSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjo0fSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6eyJQcm9kdWN0Ijp7Il9pZCI6IjI1NDYiLCJkaW1lbnNpb25zIjp7IkJhdW11c3RlclNob3J0IjpbIjI1NDYiXX19fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIzMjE1Mjg4NDgyODgxNjA4NiJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55975e3bf75a <unknown>
#1 0x55975de724b0 <unknown>
#2 0x55975de58d86 <unknown>
#3 0x55975de5769a <unknown>
#4 0x55975de57dff <unknown>
#5 0x55975de57d54 <unknown>
#6 0x55975de7669f <unknown>
#7 0x55975df1b92a <unknown>
#8 0x55975dee9592 <unknown>
#9 0x55975df0f5e0 <unknown>
#10 0x55975dee9363 <unknown>
#11 0x55975deb5d63 <unknown>
#12 0x55975deb69c1 <unknown>
#13 0x55975e384a6b <unknown>
#14 0x55975e388951 <unknown>
#15 0x55975e36cb62 <unknown>
#16 0x55975e3894c4 <unknown>
#17 0x55975e35113f <unknown>
#18 0x55975e3ad6f8 <unknown>
#19 0x55975e3ad8d6 <unknown>
#20 0x55975e3be5a6 <unknown>
#21 0x7f200069caa4 <unknown>
#22 0x7f2000729c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2ZyL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInZlaGljbGVDb2RlIjoieDI1NCIsInZlaGljbGVCTSI6IjI1NDYiLCJ2ZWhpY2xlUElIaXN0b3J5IjoiMjU0NiIsInRpbWVzdGFtcCI6MTc0NjMxNDIzMTE1NSwiZXhwaXJhdGlvbiI6MTIwfX19LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjcxNDExMzI2My4xNzQ2MzE0MjI2In0sImFub25JZCI6IjU1Yjg2NGI3OGYxMGQzY2MifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoubHUvZnIvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI1NCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiNzE1MDc5NzA2OTUzNzY3NSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55975e3bf75a <unknown>
#1 0x55975de724b0 <unknown>
#2 0x55975de58d86 <unknown>
#3 0x55975de5769a <unknown>
#4 0x55975de57dff <unknown>
#5 0x55975de57d54 <unknown>
#6 0x55975de7669f <unknown>
#7 0x55975df1b92a <unknown>
#8 0x55975dee9592 <unknown>
#9 0x55975df0f5e0 <unknown>
#10 0x55975dee9363 <unknown>
#11 0x55975deb5d63 <unknown>
#12 0x55975deb69c1 <unknown>
#13 0x55975e384a6b <unknown>
#14 0x55975e388951 <unknown>
#15 0x55975e36cb62 <unknown>
#16 0x55975e3894c4 <unknown>
#17 0x55975e35113f <unknown>
#18 0x55975e3ad6f8 <unknown>
#19 0x55975e3ad8d6 <unknown>
#20 0x55975e3be5a6 <unknown>
#21 0x7f200069caa4 <unknown>
#22 0x7f2000729c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'